implementations/python/sugar-lift-python-source/src/sugar_lift_python_source/value_pins.py

```
@@ -108,6 +108,8 @@ class ValuePin:
 
 @dataclass(frozen=True)
 class MutableGlobalPin:
+    source_cid: str
+    binding_occurrence: "SourceMemento"
```

Ruff (0.16.0)

[error] Undefined name `SourceMemento`

(F821)